src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,32 @@
 import React from 'react';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../MovieCard/MovieType';
+
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({
+  movies,
+}) => (
+  <div className="movies">
+    {movies.map(({
+      title,
+      description,
+      imgUrl,
+      imdbUrl,
+      imdbId,
+    }) => (
+      <div key={imdbId}>
+        <MovieCard
+          title={title}
+          description={description}
+          imgUrl={imgUrl}
+          imdbUrl={imdbUrl}
+        />
+      </div>
```

Comment:
в такому випадку твої карточки будуть розтягуватися на всю висоту
```suggestion
      <MovieCard
        key={imdbId}
        title={title}
        description={description}
        imgUrl={imgUrl}
        imdbUrl={imdbUrl}
      />
```